content/en/metrics/summary.md

```diff
@@ -29,18 +29,26 @@ Tag filtering supports boolean and wildcard syntax so that you can quickly ident
 
 ## Facet panel
 
+<div class="alert alert-warning">
+Query Activity are in public beta.
+</div>
+
 The search bars provide the most comprehensive set of actions to filter the list of metrics. But facets can also quickly filter your metrics by:
-* **Configuration** : Quickly identify metrics with tag configurations or additional percentile aggregations
+* **Configuration** : Quickly identify metrics with tag configurations
```

Comment:
I think we can trim these bullet points down and avoid the repetition. For example:

- **Configuration**: Metrics with tag configurations
- **Percentiles**: Distribution metrics enabled by percentiles/advanced query capabilities
- **Historical Metrics**: Disabled historical metrics
- **Query Activity**: Metrics not queried in Datadog products in the past 30 days
- **Metric Type**: Differentiate between distributions and non-distributions (counts, gauges, rates)
- **Distribution Metric Origin**: Metric origination by product (for example, metrics generated from Logs or APM Spans)